src/ax3/filters/ToString.hx

```diff
@@ -22,7 +22,7 @@ class ToString extends AbstractFilter {
 					case [TTString, TTString]:
 						e; // ok
 
-					case [TTAny, TTString]:
+					case [TTAny | TTObject(_), TTString]:
```

Comment:
So before it reported an error?

Reply:
No, it took the `case _:` case; so nothing really changed here. I just moved it explicitly into the ASAny case to also take advantage from the "handle at run-time" comment. ASAny and ASObject behave the same in this regard, so I put them together.